src/Snppts/Snippets/MbongoUI.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace Snppts.Snippets
+{
+    public class MbongoUI : IAmASnippet
+    {
+        public string Slug => "MbongoUI";
+        public string Title => "MbongoUI";
+        public string Description => "Developed in XamarinForms to show how we can develop beatifful screens with XamarinForms";
```

Comment:
Spelling mistake in beautiful

Reply:
the problem is with my keyboard, but thanks